modules/ROOT/pages/service-mesh-overview-and-landing-page.adoc

```diff
@@ -15,30 +15,33 @@ To alleviate issues arising from differences in these microservices, Anypoint Se
 
 == Business Solution
 
-Growing business demands for diverse deployment environments also introduces a need for rapidly deployable and manageable microservices architecture. In a microservice architecture, you build smaller, targeted services instead of large monolithic applications. These services are faster to build, easier to maintain, and help foster agility. 
+Growing business demands for diverse deployment environments introduces a need for rapidly deployable and manageable microservices architecture. In a microservice architecture, you build smaller, targeted services instead of large monolithic applications. These services are faster to build, easier to maintain, and help foster agility. 
 
-In a common enterprise setup, there are numerous teams building services that connect and expose functionalities across all sorts of systems: mainframes, SaaS apps, IoT devices, and so on. The result is usually a siloed environment, with teams managing these apps in different ways. 
+In a common enterprise setup, numerous teams are engaged in building services that connect and expose functionalities across all sorts of systems: mainframes, SaaS apps, IoT devices, and so on. The result is a siloed environment, with teams managing these applications in different ways. 
 
-With Anypoint Service Mesh from MuleSoft you can connect your applications, data, and devices through APIs in an applications network. You can manage, govern and secure your application network within Anypoint Platform.
+With Anypoint Service Mesh, you can connect your applications, data, and devices through APIs in an applications network. You can manage, govern, and secure your application network using Anypoint Platform.
 
-Microservices might exist outside of the MuleSoft ecosystem, all built in different languages and deployed to different environments. Anypoint Service Mesh enables you to extend your application network to any type of microservice. 
+Microservices can exist outside of the MuleSoft ecosystem, built using different languages and deployed to different environments. Anypoint Service Mesh enables you to extend your application network to any type of microservice. 
 
-Most importantly, the Anypoint Service Mesh can perform these tasks across all services, independent of the technology upon which it is built. Developers can focus on building out the business logic within. You can now discover, manage and secure any type of microservice from a central control plane in the Anypoint Platform.
+Most importantly, Anypoint Service Mesh can perform these tasks across all services, independent of the technology upon which it is built. You can now discover, manage and secure any type of microservice from a central control plane in the Anypoint Platform.
```

Comment:
You can _now_ discover > You can discover

Reply:
Done! Thanks!